src/Microsoft.OData.Edm/Schema/EdmModelBase.cs

```diff
@@ -60,6 +60,11 @@ protected EdmModelBase(IEnumerable<IEdmModel> referencedModels, IEdmDirectValueA
                 this.referencedEdmModels.Add(AuthorizationVocabularyModel.Instance);
             }
 
+            if (ValidationVocabularyModel.Instance != null)
+            {
+                this.referencedEdmModels.Add(ValidationVocabularyModel.Instance);
+            }
+

```

Comment:
I thought we weren't going to add ValidationVocabulary by default?  I would also remove AuthorizationVocabulary (I don't think anyone is using it yet).  I would also remove AlternateKeys if it hadn't already been around long enough that I was afraid people were already taking dependencies on it.

Reply:
AuthorizationVocabulary is using by our openapi library, Microsoft.OpenApi.OData. :)

Adding the model by default is to support the Edm model reading from CSDL, otherwise, it can't work. So, we should find a solution to make all the vocabulary models custom-able.